python/GangaLHCb/Lib/Applications/GaudiExec.py

```diff
@@ -342,28 +342,27 @@ def configure(self, masterappconfig):
         dir_name = self.directory
         return (None, None)
 
-
```

Comment:
The convention is supposed to be 2 lines between methods I think (this could be pep8, I just treied to copy code from elsewhere to make things more consistent)